lesson9/main.js

```diff
@@ -0,0 +1,52 @@
+//課題の配列
+const data = [
+   {to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク", src: "img/bookmark.jpeg"},
+   {to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ", src: "img/message.jpeg"}
+];
+
+const body = document.getElementById("body");
+const wrapper = document.createElement("div");
+body.appendChild(wrapper);
+
+//はじめからloading iconを表示するだけなので即時関数にしている
+(function () {
+   const loadingIcon = document.createElement("img");
+   loadingIcon.src = "img/loading-circle.gif";
+   wrapper.appendChild(loadingIcon);
+})();
+
+function createListElements() {
```

Comment:
I think it would be a good idea to fix createListElements to an arrow function as well.

Reply:
I changed createListElements() to the Arrow function.
3d108ac